pipeline/transformer/transformer_sierra/src/main/scala/uk/ac/wellcome/platform/transformer/sierra/data/SierraPhysicalItemOrder.scala

```diff
@@ -0,0 +1,88 @@
+package uk.ac.wellcome.platform.transformer.sierra.data
+
+import java.io.InputStream
+import uk.ac.wellcome.storage.streaming.Codec._
+import weco.catalogue.internal_model.identifiers.{IdState, IdentifierType}
+import weco.catalogue.internal_model.work.Item
+import weco.catalogue.source_model.sierra.{SierraBibNumber, SierraItemNumber}
+import weco.catalogue.source_model.sierra.Implicits._
+
+/** The order of physical items in Sierra is significant, and can affect
+  * how items should be requested by users.
+  *
+  * For example, you might have four items that are meant to go:
+  *
+  *     Thing you can request
+  *     Please request top item
+  *     Another thing you can request
+  *     Please request top item
+  *
+  * This "Please request top item" occurs when two items have been bound together
+  * (e.g. on the same frame) and they have to be requested together.
+  *
+  * This ordering is stored in a part of Sierra we can't access through the
```

Comment:
> we can't access

Do we have a channel to request this?

Reply:
Yeah, it's tracked by a couple of other GitHub tickets.

Reply:
Do you have a link to that API?

Reply:
https://github.com/wellcomecollection/platform/issues/4102#issuecomment-608341955

That's on a ticket linked to from the ticket in this PR description.